test/ct_test.c

```diff
@@ -124,6 +124,7 @@ static CT_TEST_FIXTURE set_up(const char *const test_case_name)
 static void tear_down(CT_TEST_FIXTURE fixture)
 {
     CTLOG_STORE_free(fixture.ctlog_store);
+    SCT_free(fixture.sct);
```

Comment:
I'm a little confused that these don't show up in CI, nor in my local build. Does test/recipes/80-test_ct.t work as intended?